common/eth2_network_config/built_in_network_configs/holesky/config.yaml

```diff
@@ -0,0 +1,118 @@
+# Extends the mainnet preset
+PRESET_BASE: 'mainnet'
+CONFIG_NAME: holesky
+
+# Genesis
+# ---------------------------------------------------------------
+# `2**14` (= 16,384)
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 16384
+# Sep-15-2023 14:55:00 +UTC
+# This is an invalid valid and should be updated when you create the genesis
```

Comment:
```suggestion
# Sep-15-2023 13:55:00 +UTC
```

Update this assuming my PR upstream gets merged: https://github.com/eth-clients/holesky/pull/66